search/base.py

```diff
@@ -44,9 +44,9 @@ class MockedESTestCase(TestCase):
     @classmethod
     def setUpClass(cls):
         cls.patchers = []
-        for name, val in indexing_api.__dict__.items():
-            if callable(val):
-                cls.patchers.append(patch('search.indexing_api.{0}'.format(name), autospec=True))
+        for name, val in tasks.__dict__.items():
+            if callable(val) and name.startswith("_"):
```

Comment:
I actually think that patching _fewer_ functions is better, if we can get away with it, so I think it's better to have the condition there. I just want to know why it was added, and what changed (if anything).